tests/suites/host_test.function

```diff
@@ -565,7 +565,11 @@ int execute_tests( int argc , const char ** argv )
 #endif /* __unix__ || __APPLE__ __MACH__ */
 
                 function_id = strtol( params[0], NULL, 10 );
-                if ( (ret = check_test( function_id )) == DISPATCH_TEST_SUCCESS )
+                if( function_id < 0 )
```

Comment:
Should we change `int function_id = 0;` to `size_t function_id = 0;`?

Reply:
My thought process was we:

1. Parse whatever integer comes in,
2. We validate the lower bound as soon as we get it, as we know its value,
3. We validate the upper bound later, because only then we know the tests count,
4. We use the integer.

However, now, when I thing about it again, I'm sure we can discard the negative integers just like any other invalid value. Thanks for the suggestion. I'll implement it.

Reply:
OK as-is. Thanks for explaining.